index.js

```diff
@@ -0,0 +1,99 @@
+const q = require('daskeyboard-applet');
+const request = require('request-promise');
+const logger = q.logger;
+
+// Check the price of a selected flight every minute in order to buy it at the best price
+// Compare the current price to the old price stored one minute ago
+
+class FlightPriceWatcher extends q.DesktopApp {
+
+	constructor() {
+		super();
+		// every minute
+		this.pollingInterval = 60 * 1000; // ms
+		logger.info("FlightPriceWatcher starting: Get the cheapest flight price!");
+	}
+
+	async getPrice() {
+		// Get the current price of the selected flight 
+		logger.info(`Getting price`);
+		const API_BASE_URL = `https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices`;
+		const API_USA_URL = 'browsequotes/v1.0/US/USD/en-US';
```

Comment:
Does that mean it's only flights in the USA ?

Reply:
Yes only for flights in USA

Reply:
It should be international. I am sure that the API allows it